### PR TITLE
updated gui and added features to configure script name etc.

### PR DESCRIPTION
**Pull request by aryangupta701**
- send the script to close the window handle
- script name can be configured 
- save script button should be disabled if no script is present
- send a ZestClientElementClear statement before every send keys to make sure it is empty
- improved gui

### CODE REVIEW

**Comment by aryangupta701** [comment]
@thc202 Please review. It has now solved the issue of multiple scripts being recorded for a single interaction as well.

File: views/options.html [hunk 1]
```diff
@@ -23,12 +23,12 @@ <h1>ZAP Browser Extension</h1>
         <td>As set in the ZAP API options</td>
       </tr>
       <tr>
-        <td><label>Enable</label></td>
+        <td><label>Enable </label></td>
```
Comment (thc202) [inline views/options.html:26]: Is this change needed?

File: views/options.html [hunk 2]
```diff
@@ -23,12 +23,12 @@ <h1>ZAP Browser Extension</h1>
...
         <td><input type="checkbox" id="zapenable"/></td>
       </tr>
-      <tr>
-        <td><label>Record</label></td>
-        <td><input type="checkbox" id="zaprecordingactive"/></td>
+      <tr >
```
Comment (thc202) [inline views/options.html:29]: Revert?

File: views/options.html [hunk 3]
```diff
@@ -23,12 +23,12 @@ <h1>ZAP Browser Extension</h1>
...
+        <td><label>Close window handle when when recording is stopped </label></td>
```
Comment (thc202) [inline views/options.html:30]: The when is duplicated but I think `Automatically close the browser in the recorded scripts.` would be more clear.

**Comment by thc202** [comment]
I no longer see the earlier recording issues but sometimes the recording does not include all interactions (e.g. misses a click), might be usage problem though.

File: views/options.html (hunk 1 above)
Comment (aryangupta701) [reply views/options.html:26]: noo will change it back

File: views/options.html (hunk 2 above)
Comment (aryangupta701) [reply views/options.html:29]: No, now we don't need it here. This was being used to start or stop the recording.

**Comment by aryangupta701** [comment]
Oh, maybe this could be a future bug fix. For me, it works fine.

File: views/options.html (hunk 2 above)
Comment (thc202) [reply views/options.html:29]: I meant the space added to the tr tag.

**Comment by thc202** [comment]
Agreed, I don't think that's a blocker.

**Review by psiinon — APPROVED** [review]

File: views/options.html (hunk 2 above)
Comment (thc202) [reply views/options.html:29]: Bump, not essential but if it's not necessary.

File: views/options.html [hunk 4]
```diff
@@ -26,9 +26,9 @@ <h1>ZAP Browser Extension</h1>
         <td><label>Enable</label></td>
         <td><input type="checkbox" id="zapenable"/></td>
       </tr>
-      <tr>
-        <td><label>Record</label></td>
-        <td><input type="checkbox" id="zaprecordingactive"/></td>
+      <tr >
```
Comment (thc202) [inline views/options.html:29]:
```suggestion
      <tr>
```

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!
